index.html

```diff
@@ -0,0 +1,459 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>NY-Times</title>
+    <link rel="stylesheet" href="styles/style.css">
+    
+</head>
+<body>
+    <header>
+        <nav>
+            <img class="left" src="assets/navsidebar.png" alt="sidebar icon">
+            <img class="left" src="assets/navsearch.png" alt="navigation search icon">
+            <p class="left" >space & cosmos</p>
+            <img class="logo middle" src="assets/nytimes-logo.png" alt="New York Times Logo">
+            <button class="right">login</button>
+            <button class="right">subscribe now</button>
+        </nav>
+    </header>
+    <main>
+        <article class="first-article">
+        <section class="section1">
+        <h1 id="section-title">Space Ripples Reveal Big Bang’s Smoking Gun</h1>
```

Comment:
### `REQUIRED`

Consider using `h2-h6` element here instead. `<h1>` should be used only as a top-level heading. 

#### :bulb:  Note that `<h1>` is nested inside three sectional elements here: `main > article > section > h1`.

> Using only h1 elements in a HTML document results in a flat document outline as heading element semantics are conveyed to users as per the numeric in the heading element tag name.

Learn more [here](https://www.w3.org/wiki/HTML/Usage/Headings/h1only)